app/widget.js

```diff
@@ -60,14 +60,14 @@ function renderWidget(el) {
         data: window._SimplechartWidgetData[el.id],
       }));
     }
-
-    // And listen for 'widgetData' event triggered on widget element
-    listenerWidgetData(el, store.dispatch);
   } else {
-    // Bye.
+    // Must have either data-url or data-var
```

Comment:
Not sure what I want to do with an error since I don't want to break other charts on the page if one has a malformed embed code. But I _definitely_ want to stop `renderWidget()` from executing if it can't find `data-var` or `data-url`